db/migrate/20230213160135_delete_receipt_printing_from_preferences.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteReceiptPrintingFromPreferences < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
```suggestion
  def up
```

Otherwise the delete will also happen on rollback.